packages/card/src/react/index.js

```diff
@@ -9,77 +9,114 @@ import { sizes as iconSizes } from '@pluralsight/ps-design-system-icon'
 import filterReactProps from '@pluralsight/ps-design-system-filter-react-props'
 import { useTheme } from '@pluralsight/ps-design-system-theme'
 
-import css from '../css/index.js'
+import stylesheet from '../css/index.js'
 import { toPercentageString } from '../js/index.js'
 import * as vars from '../vars/index.js'
 
 if (typeof window !== 'undefined') polyfillFocusWithin(document)
 
 const styles = {
-  actionBar: ({ actionBarVisible: visible, fullOverlay }) =>
-    glamor.css({
-      ...css['.psds-card__action-bar'],
-      ...(fullOverlay &&
+  actionBar: ({ actionBarVisible: visible, fullOverlay }) => {
+    const label = 'psds-card__action-bar'
```

Comment:
i used a variable to store the base class names in a few places to make refactoring easier. i used the new label because in the future we might use the labels feature of glamor like so


> css({ label })